src/lib/y2users/linux/reader.rb

```diff
@@ -19,12 +19,101 @@
 
 require "yast2/execute"
 require "y2users/linux/base_reader"
+require "y2users/login_config"
+require "y2users/linux/useradd_config_reader"
+require "users/ssh_authorized_keyring"
+
+Yast.import "Autologin"
+Yast.import "MailAliases"
 
 module Y2Users
   module Linux
     # Reads users configuration from the system using `getent` command.
     class Reader < BaseReader
-    private # rubocop:disable Layout/IndentationWidth
+      # @see BaseReader#read
+      # @note some of these #read_* methods could be moved to the
+      #   {BaseReader#read} once they allow reading from a specific location
+      #   (i.e., compatible with {LocalReader} too)
+      def read
+        config = super
+
+        read_root_aliases(config)
+        read_home_permissions(config)
+        read_authorized_keys(config)
+        read_useradd_config(config)
+        read_login(config)
+
+        config
+      end
+
+    private
+
+      # Set root aliases (i.e., users receiving system mails)
+      #
+      # @see Yast::MailAliases#GetRootAlias
+      #
+      # @param config [Config]
+      def read_root_aliases(config)
+        Yast::MailAliases.GetRootAlias.split(", ").each do |name|
+          user = config.users.by_name(name)
+          user.receive_system_mail = true if user
+        end
+      end
+
+      # Command for reading home directory permissions
+      STAT = "/usr/bin/stat".freeze
+      private_constant :STAT
+
+      # Reads home permissions
+      #
+      # @return [Array<Y2Users::User>]
+      def read_home_permissions(config)
+        config.users.reject(&:system?).each do |user|
+          next unless user.home && Dir.exist?(user.home.path)
+
+          # NOTE: '#' printf flag is needed to make sure permissions bits
```

Comment:
I would simply this:

~~~
# "stat --printf %#a" returns the permissions in octal format (e.g., 0755), as expected by {Home#permissions}. 
~~~